packages/back-end/src/jobs/sdkWebhooks.ts

```diff
@@ -393,7 +409,24 @@ export async function fireGlobalSdkWebhooks(
     const payload = JSON.stringify(defs);
 
     WEBHOOKS.forEach((webhook) => {
-      const { url, signingKey, method, headers, sendPayload } = webhook;
+      const {
+        url,
+        signingKey,
+        method,
+        headers,
+        sendPayload,
+        payloadFormat,
+      } = webhook;
+      let format = payloadFormat;
+      if (!format) {
+        if (method === "GET") {
+          format = "none";
+        } else if (sendPayload) {
+          format = "standard";
+        } else {
+          format = "standard-no-payload";
+        }
```

Comment:
Aren't we guaranteeing that this never happens b/c of the migration in the interface conversion function?

Reply:
yeah, we can remove this

Reply:
actually no, we need this. This is for global webhooks which are set via env var. The interface conversion is only for standard webhook models.